modules/cli-installing-cli-brew.adoc

```diff
@@ -15,9 +15,18 @@ For macOS, you can install the OpenShift CLI (`oc`) by using the link:https://br
 
 .Procedure
 
-* Run the following command to install the link:https://formulae.brew.sh/formula/openshift-cli[openshift-cli] package:
+* Install the link:https://formulae.brew.sh/formula/openshift-cli[openshift-cli] package by running the following command:
 +
 [source,terminal]
 ----
 $ brew install openshift-cli
```

Comment:
I know this is outside the scope of change, but `brew` does not have an `install` option like is presented here.